homeassistant/components/broadlink/climate.py

```diff
@@ -71,7 +71,8 @@ async def async_set_temperature(self, **kwargs: Any) -> None:
     @callback
     def _update_state(self, data: dict[str, Any]) -> None:
         """Update data."""
-        self.sensor_mode = SensorMode(data.get("sensor"))
+        if data.get("sensor") is not None:
```

Comment:
```suggestion
        if data.get("sensor"):
```
To align with the codestyle of the lines below

Reply:
Disagreed. I need a None check. The lines below are 0 check, so it would be false if sensor is 0.